templates/todo/common/infra/bicep/app/api-appservice-dotnet.bicep

```diff
@@ -10,7 +10,7 @@ param appSettings object = {}
 param keyVaultName string
```

Comment:
Please include a bicepconfig.json file like we have here: https://github.com/Azure/azure-dev/pull/1251/files

Reply:
done